dlrs/main/classes/RollupService.cls

```diff
@@ -1235,9 +1235,7 @@ global with sharing class RollupService {
         }
         Map<String, RollupSummary> shadowRollupsByMdtId = new Map<String, RollupSummary>();
         for (
-          RollupSummary shadowRollup : new RollupSummariesSelector.CustomObjectSelector(
```

Comment:
This change allows accessing the data cache instead of re-querying the records. Added a new public method to the class to allow requesting records that aren't active.